test/test_utils/test_reporting.py

```diff
@@ -61,6 +61,7 @@ class TestReportGenerator:
         "test_ecs_tensorflow_training_mnist_gpu",
         "test_ecs_tensorflow_inference_gpu",
         "test_ecs_tensorflow_inference_gpu_nlp",
+        "test_ecs_pytorch_inference_gpu",
```

Comment:
I think this had been missing for quite a long time. Thanks for adding it! 

How did we realize the need to add it?

Reply:
ecs tests were failing because it took a while to terminate `p3.8xlarge` instances. I changed it to `g4dn.8xlarge` to mirror Sai's changes on TF tests. This then threw an error in `dlc-pr-quick-checks` because it expected the tests to run on multi gpu instance types.